NeoMathEngine/include/NeoMathEngine/NeoMathEngineDefs.h

```diff
@@ -49,6 +49,7 @@ limitations under the License.
 
 #include <climits>
 #include <cstddef>
+#include <algorithm>
```

Comment:
This header belongs to external API, better add it to another place.
P.S. I'm not sure about __any__ includes here...

Reply:
additional include removed.